libs/community/langchain_community/document_loaders/spider.py

```diff
@@ -45,10 +51,6 @@ def __init__(
         if params is None:
```

Comment:
this line is now not needed right? this is the same check as what is on line 34

Reply:
Yep that is correct, will remove it in 10 mins

Reply:
It is now removed 👍